### PR TITLE
Add default recipes

### PR DESCRIPTION
**Pull request by rcrichton**
Changes:

* Modify release action
* Change mapping medaitor config to allow JemPI to work with MPI mediator
* Update Jempi package to use latest code and new Keycloak API
* Update default config in package to create a unified example config that can be used to show how everything works together

Test using:
* `instant package init -p cdr-dw --dev`
* `instant package init -p cdr --dev`
* `instant package init -p mpi --dev`

For more detail see docs - https://jembi.gitbook.io/openhim-platform/recipes/central-data-repository-with-data-warehousing

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 projectName: platform
-image: jembi/platform:2.5.0
-logPath: /tmp/logs
+image: jembi/platform:test-recipes
```
Comment (marrouchi) [inline config.yaml:2]: Are we sure we want to set this tag ?

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 1 above)
Comment (rcrichton) [reply config.yaml:2]: Oops, nope. That should be changed.

Comment (rcrichton) [reply config.yaml]: Fixed

**Comment by bradsawadye** [comment]
@rcrichton it looks good. If we are going to merge this in, we will need to change the docker images wth the "test" tags. The endpoint for querying patients is now broken

**Comment by rcrichton** [comment]
Thanks Brad. I need to wait for a release of JemPI for that. I think they are putting out on soon so I will hold off for that.

**Review by bradsawadye — APPROVED** [review]

File: analytics-datastore-clickhouse/importer/config/clickhouseTables.js [hunk 2]
```diff
@@ -1,22 +1,52 @@
-const CLUSTERED_MODE = process.env.CLUSTERED_MODE || 'true';
+const CLUSTERED_MODE = process.env.CLUSTERED_MODE || "true";
 
 const queries =
   Boolean(CLUSTERED_MODE) === true
```
Comment (marrouchi) [inline analytics-datastore-clickhouse/importer/config/clickhouseTables.js:4]: We should invert the condition here

File: interoperability-layer-openhim/package-metadata.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
     "OPENHIM_CONSOLE_CPU_RESERVE": "0.05",
     "OPENHIM_CONSOLE_MEMORY_LIMIT": "2G",
     "OPENHIM_CONSOLE_MEMORY_RESERVE": "500M",
+    "OPENHIM_CONSOLE_SHOW_LOGIN": "true",
```
Comment (marrouchi) [inline interoperability-layer-openhim/package-metadata.json:22]:
```suggestion
    "OPENHIM_CONSOLE_SHOW_LOGIN": "false",
```
Is keycloak enabled by default ?

File: client-registry-jempi/package-metadata.json [hunk 4]
```diff
@@ -67,12 +70,12 @@
     "JEMPI_POSTGRES_MEMORY_LIMIT": "3G",
     "JEMPI_POSTGRES_MEMORY_RESERVE": "500M",
     "JEMPI_SESSION_SECURE": false,
+    "JEMPI_SESSION_DOMAIN_NAME": "localhost",
     "DOMAIN_NAME": "",
     "POSTGRESQL_PASSWORD": "postgres",
```
Comment (marrouchi) [inline client-registry-jempi/package-metadata.json:75]: POSTGRESQL_USERNAME and POSTGRESQL_DATABASE are missing here right ?

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: interoperability-layer-openhim/package-metadata.json (hunk 3 above)
Comment (bradsawadye) [reply interoperability-layer-openhim/package-metadata.json:22]: I think its fine to leave as is. These configs will be overridden by whats in the env files for the recipes

File: client-registry-jempi/package-metadata.json (hunk 4 above)
Comment (bradsawadye) [reply client-registry-jempi/package-metadata.json:75]: The variables are in there

Comment (bradsawadye) [reply client-registry-jempi/package-metadata.json:75]: https://github.com/jembi/platform/pull/261/files/5cbc541df9c0f3cce1a753663341bd745d99763e#diff-fc2d1132260ae5045f50e2d801eea9b429149663100c842d595a5de7c020bd8dR63-R64

File: analytics-datastore-clickhouse/importer/config/clickhouseTables.js (hunk 2 above)
Comment (bradsawadye) [reply analytics-datastore-clickhouse/importer/config/clickhouseTables.js]: done

**Review by marrouchi — APPROVED** [review]

**Pull request merged**